set.mm

```diff
@@ -491222,60 +491647,62 @@ of the indicator function (Contributed by Thierry Arnoux,
+      -> 0 < ( # ` ( ( O i^i Prime ) ( repr ` 3 ) N ) ) ) $=
       ( vn cfv co cc0 cprime cin c3 crepr chash cn wcel clt wbr c0 tgoldbachgnn
       cfn wne nnnn0d cn0 3nn0 a1i inss2 prmssnn sstri reprfi2 wceq cv cvma cmul
       wss c1 c2 tgoldbachgtde gt0ne0d neneqd wa simpr sumeq1d sum0 syl6eq mtand
       csu neqned hashnncl biimpar syl2anc nngt0 syl ) AHUAUBZGUCUDRSZUERZUFUGZT
       WGUHUIAWFULUGZWFUJUMZWHAWEUCGAGACGHIJKUKUNUCUOUGAUPUQWEUFVFAWEUAUFHUAURUS
       UTUQVAAWFUJAWFUJVBZWFTQVCZRZVDRWMERVESVGWLRZVDRWNFRVESVHWLRZVDRWOFRVESVES
       VESZQVRZTVBAWQTAWQABCDQEFGHIJKLMNOPVIVJVKAWKVLZWQUJWPQVRTWRWFUJWPQAWKVMVN
-      WPQVOVPVQVSWIWHWJWFVTWAWBWGWCWD @.
-  @}
-
-  @{
-    @d N h k x y z @.  @d O h k z @.  @d h k m x y z @.  @d h k x y ph @.
-    tgoldbachgtd.o @e |- O = { z e. ZZ | -. 2 || z } @.
-    tgoldbachgtd.n @e |- ( ph -> N e. O ) @.
-    tgoldbachgtd.1 @e |- ( ph -> ( ; 1 0 ^ ; 2 7 ) <_ N ) @.
-    @( Odd integers greater than ` ( ; 1 0 ^ ; 2 7 ) ` have at least a
-       representation as a sum of 3 odd primes.  Final statement in section 7.4
-       of [Helfgott] p. 70 (Contributed by Thierry Arnoux, 15-Dec-2021.) @)
-    tgoldbachgtd @p |- ( ph -> 0 < ( # ` ( ( O i^i Prime ) ( repr ` 3 ) N ) ) )
-      @=
-      ( vm cv cfv c1 cc0 cdp2 co cle wbr c2 cexp cmul cn vk vh vx vy cabs c7 c9
-      c5 cdp cn0 wral c4 c8 cioo cvma cof cvts ci cpi cneg ce w3a cprime cin c3
-      citg crepr chash clt cpnf cico cmap wcel wa ad3antrrr cdc elmapi ad3antlr
-      wf ad2antlr simpr1 r19.21bi simpr2 simpr3 wceq fveq2 oveq1d oveq12d oveq2
-      oveq2d fveq2d cbvitgv syl6breq tgoldbachgtda hgt749d r19.29vva ) AHIZUAIZ
-      JUEJKLUFUGUGUHUHMMMMMUINOPZHUJUKZWQUBIZJUEJKULKULMMUINOPZHUJUKZLLLLULQQUL
-      UMMMMMMMMUINCQRNSNZUCLKUNNZUCIZUOXASUPZNCUQNZJZXFUOWRXGNCUQNZJZQRNZSNZURQ
-      USSNSNZCUTZXFSNZSNZVAJZSNZVFZOPZVBZLDVCVDCVEVGJNVHJVIPUBUALVJVKNZTVLNZYDA
-      XAYDVMZVNZWRYDVMZVNZYBVNZUDBHXAWRCDEACDVMYEYGYBFVOAKLVPQUFVPRNCOPYEYGYBGV
-      OYETYCXAVSAYGYBXAYCTVQVRYGTYCWRVSYFYBWRYCTVQVTYIWSHUJYHWTXCYAWAWBYIXBHUJY
-      HWTXCYAWCWBYIXDXTUDXEUDIZXHJZYJXJJZQRNZSNZXNXOYJSNZSNZVAJZSNZVFOYHWTXCYAW
-      DUCUDXEXSYRXFYJWEZXMYNXRYQSYSXIYKXLYMSXFYJXHWFYSXKYLQRXFYJXJWFWGWHYSXQYPV
-      AYSXPYOXNSXFYJXOSWIWJWKWHWLWMWNAUCBUBUAHCDEFGWOWP @.
-  @}
+      WPQVOVPVQVSWIWHWJWFVTWAWBWGWCWD $.
+  $}
 
-  @{
-    @d G m @.  @d O c i m p q r z @.  @d c i m n p q r z @.
-    tgoldbachgt.o @e |- O = { z e. ZZ | -. 2 || z } @.
-    tgoldbachgt.g @e |- G = { z e. O | E. p e. Prime E. q e. Prime
+  ${
+    $d N h k m n x y z $.  $d O h k m n z $.  $d h k m n x y ph $.
+    tgoldbachgtd.o $e |- O = { z e. ZZ | -. 2 || z } $.
+    tgoldbachgtd.n $e |- ( ph -> N e. O ) $.
+    tgoldbachgtd.1 $e |- ( ph -> ( ; 1 0 ^ ; 2 7 ) <_ N ) $.
+    $( Odd integers greater than ` ( ; 1 0 ^ ; 2 7 ) ` have at least a
+       representation as a sum of three odd primes.  Final statement in section
+       7.4 of [Helfgott] p. 70 (Contributed by Thierry Arnoux, 15-Dec-2021.) $)
```

Comment:
Final period, and capitalized "Section" (same in following comment). (I learnt from my advisor that we generally write "the first section", but "Section 1".)